engines/account/app/workers/account/signup_worker.rb

```diff
@@ -0,0 +1,13 @@
+module Account
+  class SignupWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low
+
+    def perform(signup_id)
+      signup = Signup.find_by(id: signup_id)
+
+      SignupMailer.confirm_email(signup).deliver
```

Comment:
What I don't like and it's not completely clear to me about `ActiveJob` is if it generates the HTML even when we just want to enqueue it, and generate the HTML later.

Needs more investigation.

I agree with upgrading to 4.2 though